client/agent-opentracing/src/main/java/org/hawkular/apm/agent/opentracing/OpenTracingManager.java

```diff
@@ -57,7 +58,20 @@
 
     private static long expiryInterval = 60000;
 
+    // allow access from test
+    protected static HashSet fileExtensionWhitelist = new HashSet();
+
     static {
+        String whitelist = PropertyUtil.getProperty(PropertyUtil.HAWKULAR_APM_AGENT_FILE_EXTENSION_WHITELIST);
```

Comment:
To save you having to unnecessarily define the env var for jsp, could you add a second parameter with a default value for "jsp".